central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -190,14 +193,36 @@ func (m *managerImpl) processRequestToSensor(ctx context.Context, scanRequest *s
 		profiles = append(profiles, profile.GetProfileName())
 	}
 
-	// Check if any existing cluster that has the scan configuration with any of profiles being referenced by the scan request.
+	// Check if there any existing cluster that has the scan configuration with any of profiles being referenced by the scan request.
 	// If so, then we cannot create the scan configuration.
 	err := m.scanSettingDS.ScanConfigurationProfileExists(ctx, scanRequest.GetId(), profiles, clusters)
 	if err != nil {
 		log.Error(err)
 		return nil, errors.Wrapf(err, "Unable to create scan configuration named %q.", scanRequest.GetScanConfigName())
 	}
 
+	// check if cluster is selected
+	if len(clusters) == 0 {
+		return nil, errors.Errorf("No clusters selected for scan configuration named %q.", scanRequest.GetScanConfigName())
+	}
```

Comment:
This shouldn't occur because it is checked immediately in the service layer.  If we want to have this check in the manager, we should put it as one of the first things we do.  There is no point in doing any of the other checks if the request is bad.  Make sense?